ccdproc/tests/test_combiner.py

```diff
@@ -393,3 +393,23 @@ def test_writeable_after_combine(ccd_data, tmpdir, comb_func):
     ccd2 = getattr(combined, comb_func)()
     # This should not fail because the resulting uncertainty has a mask
     ccd2.write(tmp_file.strpath)
+
+def test_iraf_minmax_masking():
+    ccdlist = [CCDData(np.ones((3, 5))*90., unit="adu"),\
```

Comment:
If there is some open bracket (``()``, ``{}`` or ``[]``) there is no need to use ``\`` at the end of the line.